client/components/organization-wrapper/organization-wrapper.js

```diff
@@ -112,208 +116,213 @@ export default class OrganizationWrapper extends React.Component {
-                  }}
-                />
-                <Route
-                  path={`${match.path}/payment/:result`}
-                  render={(props) => {
-                    const {result} = props.match.params;
-                    return (
+                    }}
+                  />
+                  <Route
+                    render={() => (
                       <Suspense fallback={<Loader full={false} />}>
-                        <PaymentStatus cookies={cookies} result={result} />
+                        <ConnectedDoesNotExist />
                       </Suspense>
-                    );
-                  }}
-                />
-                <Route
-                  render={() => (
-                    <Suspense fallback={<Loader full={false} />}>
-                      <ConnectedDoesNotExist />
-                    </Suspense>
-                  )}
-                />
-              </Switch>
-              <Route path={match.path} render={() => <Footer />} />
-            </div>
-            <Helmet>
-              <title>
-                {pageTitle === undefined
-                  ? t`DEFAULT_TITL - ${orgName}`
-                  : pageTitle}
-              </title>
-            </Helmet>
-            {cssPath && orgSlug ? (
-              <Helmet>
-                <link rel="stylesheet" href={getAssetPath(orgSlug, cssPath)} />
-              </Helmet>
-            ) : null}
-            {favicon && orgSlug ? (
+                    )}
+                  />
+                </Switch>
+                <Route path={match.path} render={() => <Footer />} />
+              </div>
               <Helmet>
-                <link
-                  rel="shortcut icon"
-                  type="image/x-icon"
-                  href={getAssetPath(orgSlug, favicon)}
-                />
+                <title>
+                  {pageTitle === undefined
+                    ? t`DEFAULT_TITL - ${orgName}`
+                    : pageTitle}
+                </title>
               </Helmet>
-            ) : null}
-            {loading && (
-              <div className="loader-container">
-                <div className="loader" />
-              </div>
-            )}
-          </LoadingContext.Provider>
+              {loading && (
+                <div className="loader-container">
+                  <div className="loader" />
+                </div>
+              )}
+            </LoadingContext.Provider>
+          ) : null}
+          {cssPath && orgSlug ? (
+            <Helmet>
+              <link rel="stylesheet" href={getAssetPath(orgSlug, cssPath)} />
+            </Helmet>
+          ) : null}
+          {favicon && orgSlug ? (
+            <Helmet>
+              <link
+                rel="shortcut icon"
+                type="image/x-icon"
+                href={getAssetPath(orgSlug, favicon)}
+              />
+            </Helmet>
+          ) : null}
         </>
-      ) : null;
+      );
```

Comment:
In this the only change is rearrangement i.e. moving `cssPath` block and `favicon` block outside the translation loaded block so that re-render of the page with CSS files does not take place.